lib/fortnox/api/models/row/entity.rb

```diff
@@ -0,0 +1,66 @@
+require "fortnox/api/models/entity/base"
+
+module Fortnox
+  module API
+    module Entities
+      class Row < Fortnox::API::Entities::Base
```

Comment:
If I should keep it in models, what module should I use? The current module is a bit misleading since it's not an entity: `Fortnox::API::Entities`. Also, the class itself is named `entity.rb`. Maybe we could use `Fortnox::API::Models` and place it directly in at `lib/fortnox/api/models/row.rb`.